js/audio/static/StaticAudio.svelte

```diff
@@ -43,9 +38,9 @@
 	<div class="icon-buttons">
 		{#if show_download_button}
 			<a
-				href={value.data}
```

Comment:
@pngwn here we're using `url` but in `VideoPreview` we're using `data`. Which is intended?